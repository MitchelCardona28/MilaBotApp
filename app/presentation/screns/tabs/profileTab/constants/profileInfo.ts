import  ProfileSchema  from "../schemas/profileSchema"

export const ProfileInfo = {
    name: {
        title: 'Nombre',
        icon: 'clipboard-account-outline',
        color: 'experianPurple',
    },
    lastName: {
        title: 'Apellido',
        icon: 'clipboard-account-outline',
        color: 'experianPink',
    },
    dni: {
        title: 'Nro. de IdentificacioÃÅn',
        icon: 'card-account-details-outline',
        color: 'experianBlue',
    },
    email: {
        title: 'Email',
        icon: 'email',
        color: 'experianPink',
    },
    phone: {
        title: 'Celular',
        icon: 'cellphone',
        color: 'experianPurple',
    }
}



export const userData: ProfileSchema = {
    name: 'Mila',
    lastName: 'Bot',
    email: 'mila.bot@me.com',
    phone: '+569 12345678',
    dni: '12345678'
}