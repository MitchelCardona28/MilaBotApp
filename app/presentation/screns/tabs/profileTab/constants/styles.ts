import { StyleSheet } from "react-native";

export const styles = StyleSheet.create({
    container: {
        padding: 0,
        flexGrow: 1,
        flexShrink: 1,
        flexBasis: 0,
    },
    profileAvatarWrapper: {
        position: 'relative',
    },
    profileAction: {
        position: 'absolute',
        right: -4,
        bottom: -10,
        alignItems: 'center',
        justifyContent: 'center',
        width: 30,
        height: 30,
        borderRadius: 100,
    },
    profileName: {
        marginTop: 20,
        fontSize: 19,
        fontWeight: '600',
        color: '#414d63',
        textAlign: 'center',
    },
    profileAccountName: {
        marginTop: 5,
        fontSize: 16,
        color: '#989898',
        textAlign: 'center',
    },
    section: {
        paddingHorizontal: 24,
    },
    sectionTitle: {
        paddingVertical: 12,
        fontSize: 12,
        fontWeight: '600',
        color: '#9e9e9e',
        textTransform: 'uppercase',
        letterSpacing: 1.1,
    },
    row: {
        flexDirection: 'row',
        alignItems: 'center',
        justifyContent: 'flex-start',
        height: 50,
        backgroundColor: '#f2f2f2',
        borderRadius: 8,
        marginBottom: 12,
        paddingLeft: 12,
        paddingRight: 12,
    },

});

