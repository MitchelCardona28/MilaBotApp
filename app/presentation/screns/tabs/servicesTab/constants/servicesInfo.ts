
export const services = [{
    id: 1,
    image: require('../assets/busca.png'),
    name: 'buscacrédito',
    description: 'Oportunidades financieras según tu perfil de crédito',
    color: 'experianViolet'
},
{
    id: 2,
    image: require('../assets/ponteAlDia.png'),
    name: 'ponte al día',
    description: 'Ponte al día con el pago de tus deudas',
    color: 'experianPink'
},
{
    id: 3,
    image: require('../assets/verifica.png'),
    name: 'verifica',
    description: 'Conoce tu riesgo de robo de identidad',
    color: 'experianBlue'
},
{
    id: 4,
    image: require('../assets/business.png'),
    name: 'ahorros',
    description: 'Programa el futuro de tus finanzas',
    color: 'experianPurple'
}
]