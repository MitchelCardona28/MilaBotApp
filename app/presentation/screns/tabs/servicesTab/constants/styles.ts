import { StyleSheet } from "react-native";

export const styles = StyleSheet.create({
    cardContainer: {
        margin: 3,
        
    },
    cardImage: {
        width: 85,
        height: 85,
        resizeMode: 'cover', // 'objectFit' no es una propiedad válida en React Native, usa 'resizeMode'
    },
    cardLogo: {
        width: 130, // Define un ancho para el logo
        height: 130, // Define una altura para el logo
        resizeMode: 'contain', // Asegúrate de que el logo se ajuste dentro de las dimensiones sin deformarse
        marginLeft: 10, // Agrega un margen a la izquierda para separarlo del texto
    },
    cardContent: {
        flexDirection: 'row',
        padding: 5, // Agrega un relleno al contenido de la tarjeta
    },
    nameLogoContainer: {
        flexDirection: 'row', // Organiza el nombre y el logo horizontalmente
        alignItems: 'center', // Centra los elementos verticalmente en su contenedor
    },
    cardTextContainer: {
        flexDirection: 'row',
        alignItems: 'center',
        justifyContent: 'space-between', // Ajusta esto según necesites
    },
    cardTextTitle: {
        flexDirection: 'row',
        alignItems: 'center',
        marginLeft: 10,
         
    },
    bodylarge: {
        
    },
    cardTitle: {

    }
});
