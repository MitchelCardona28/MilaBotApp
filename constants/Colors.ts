/**
 * Below are the colors that are used in the app. The colors are defined in the light and dark mode.
 * There are many other ways to style your app. For example, [Nativewind](https://www.nativewind.dev/), [Tamagui](https://tamagui.dev/), [unistyles](https://reactnativeunistyles.vercel.app), etc.
 */
import { MD3LightTheme as DefaultTheme, MD3LightTheme as DarkTheme } from 'react-native-paper';

const tintColorLight = '#0a7ea4';
const tintColorDark = '#fff';
const experianPurpleLight = 'rgb(142, 60, 150)';
const experianVioletLight = 'rgb(107, 79, 162)';
const experianBlueLight = 'rgb(78, 87, 169)';
const experianPinkLight = 'rgb(174, 21, 132)';
const experianPurpleDark = 'rgb(189, 194, 255)';
const experianVioletDark = 'rgb(211, 187, 255)';
const experianBlueDark = 'rgb(253, 169, 255)';
const experianPinkDark = 'rgb(255, 174, 219)';

export const Colors = {
  light: {
    text: '#11181C',
    background: '#fff',
    tint: tintColorLight,
    icon: '#687076',
    tabIconDefault: '#687076',
    tabIconSelected: tintColorLight,
    experianBlue: experianBlueLight,
    experianViolet: experianVioletLight,
    experianPurple: experianPurpleLight,
    experianPink: experianPinkLight
  },
  dark: {
    text: '#ECEDEE',
    background: '#151718',
    tint: tintColorDark,
    icon: '#9BA1A6',
    tabIconDefault: '#9BA1A6',
    tabIconSelected: tintColorDark,
    experianBlue: experianBlueDark,
    experianViolet: experianVioletDark,
    experianPurple: experianPurpleDark,
    experianPink: experianPinkDark
  },
};


export const PaperTheme = {
  light: {
    ...DefaultTheme,
    // Specify custom property
    myOwnProperty: true,
    // Specify custom property in nested object
    colors: {
      ...DefaultTheme.colors,
      experianBlue: experianBlueLight,
      onExperianBlue: "rgb(255, 255, 255)",
      experianBlueContainer: "rgb(224, 224, 255)",
      onExperianBlueContainer: "rgb(2, 8, 101)",
      experianViolet: experianVioletLight,
      onExperianViolet: "rgb(255, 255, 255)",
      experianVioletContainer: "rgb(234, 221, 255)",
      onExperianVioletContainer: "rgb(37, 0, 90)",
      experianPurple: experianPurpleLight,
      onExperianPurple: "rgb(255, 255, 255)",
      experianPurpleContainer: "rgb(255, 214, 251)",
      onExperianPurpleContainer: "rgb(54, 0, 61)",
      experianPink: experianPinkLight,
      onExperianPink: "rgb(255, 255, 255)",
      experianPinkContainer: "rgb(255, 216, 234)",
      onExperianPinkContainer: "rgb(60, 0, 43)"
    },
  },
  dark: {
    ...DarkTheme,
    // Specify custom property
    myOwnProperty: true,
    // Specify custom property in nested object
    colors: {
      ...DarkTheme.colors,
      experianBlue: experianBlueDark,
      onExperianBlue: "rgb(30, 38, 120)",
      experianBlueContainer: "rgb(54, 62, 144)",
      onExperianBlueContainer: "rgb(224, 224, 255)",
      experianViolet: experianVioletDark,
      onExperianViolet: "rgb(59, 29, 113)",
      experianVioletContainer: "rgb(82, 54, 137)",
      onExperianVioletContainer: "rgb(234, 221, 255)",
      experianPurple: experianPurpleDark,
      onExperianPurple: "rgb(88, 0, 99)",
      experianPurpleContainer: "rgb(115, 34, 124)",
      onExperianPurpleContainer: "rgb(255, 214, 251)",
      experianPink: experianPinkDark,
      onExperianPink:"rgb(97, 0, 72)",
      experianPinkContainer: "rgb(136, 0, 102)",
      onExperianPinkContainer: "rgb(255, 216, 234)"
    },
  }

}