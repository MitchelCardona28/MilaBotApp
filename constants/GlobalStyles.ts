import { Platform, StyleSheet } from "react-native";

export const globlalStyles = StyleSheet.create({

    safeAreaStyle: {
        padding: 0,
        flexGrow: 1,
        flexShrink: 1,
        flexBasis: 0,
    
    },
    padding: {
        padding: 10
    },
    justifyCenter: {
        display: "flex",
        alignItems: "center",
        justifyContent: "center",
    }
})