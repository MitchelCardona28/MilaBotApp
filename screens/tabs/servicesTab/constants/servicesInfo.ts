
export const services = [{
    id: 1,
    name:  require('../assets/buscaLogo.png'),
    description: 'Oportunidades financieras según tu perfil de crédito',
    image: require('../assets/busca.png'),
    color: 'experianViolet'
},
{
    id: 2,
    name:require('../assets/ponteAlDiaLogo.png'),
    description: 'Ponte al día con el pago de tus deudas',
    image: require('../assets/ponteAlDia.png'),
    color: 'experianPink'
},
{
    id: 3,
    name:  require('../assets/verificaLogo.png'),
    description: 'Conoce tu riesgo de robo de identidad',
    image: require('../assets/verifica.png'),
    color: 'experianBlue'
},
{
    id: 4,
    name:  require('../assets/businessLogo.png'),
    description: 'Programa el futuro de tus finanzas',
    image: require('../assets/business.png'),
    color: 'experianPurple'
}
]