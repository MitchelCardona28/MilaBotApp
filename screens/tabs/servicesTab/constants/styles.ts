import { StyleSheet } from "react-native";

export const styles = StyleSheet.create({

    cardContainer: {
        margin: 20
    },
    cardImage: {
        width: 100,
        height: 100,
        objectFit:'cover'
    },
    cardLogo: {
        marginBottom: 10,
        maxWidth: 150,
        objectFit:'fill'
    },
    cardContent: {
        display: "flex",
        flexDirection: "row",
        alignItems: "center",
        flexWrap: "wrap",
        gap: 20,
    },

});

