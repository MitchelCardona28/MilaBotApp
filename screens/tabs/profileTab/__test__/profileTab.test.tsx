import React from 'react';
import renderer from 'react-test-renderer';
import { Provider as PaperProvider } from 'react-native-paper';
import ProfileTab from '../index';
import { ProfileInfo, userData } from '../constants/profileInfo';
import { PaperTheme } from '@/constants/Colors';

const theme = PaperTheme.light;
describe('ProfileTab', () => {
    it('should render the profile section correctly', () => {
        const tree = renderer.create(
            <PaperProvider theme={theme}>
                <ProfileTab />
            </PaperProvider>
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });


    it('should contain profile name and account name', () => {
        const tree = renderer.create(
            <PaperProvider theme={theme}>
                <ProfileTab />
            </PaperProvider>
        ).root;

        expect(tree.findByProps({ children: 'Mi Perfil' })).toBeTruthy();
        expect(tree.findByProps({ children: 'Usuario MilaBot' })).toBeTruthy();
    });

    it('should render the contact information correctly', () => {
        const tree = renderer.create(
            <PaperProvider theme={theme}>
                <ProfileTab />
            </PaperProvider>
        ).root;

        Object.entries(ProfileInfo).forEach(([key, info]) => {
            expect(tree.findByProps({ children: info.title })).toBeTruthy();
            expect(tree.findByProps({ children: userData[key] })).toBeTruthy();
        });
    });
});
