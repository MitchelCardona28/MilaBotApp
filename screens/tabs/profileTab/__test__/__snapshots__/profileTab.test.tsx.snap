// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileTab should render the profile section correctly 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <View
    collapsable={false}
    pointerEvents="box-none"
    style={
      {
        "flex": 1,
      }
    }
  >
    <View
      style={
        {
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
          "padding": 0,
        }
      }
    >
      <View
        style={
          [
            {
              "alignItems": "center",
              "display": "flex",
              "justifyContent": "center",
            },
            {
              "padding": 24,
            },
          ]
        }
      >
        <View
          accessibilityState={
            {
              "busy": undefined,
              "checked": undefined,
              "disabled": undefined,
              "expanded": undefined,
              "selected": undefined,
            }
          }
          accessibilityValue={
            {
              "max": undefined,
              "min": undefined,
              "now": undefined,
              "text": undefined,
            }
          }
          accessible={true}
          collapsable={false}
          focusable={false}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "opacity": 1,
            }
          }
        >
          <View
            style={
              {
                "position": "relative",
              }
            }
          >
            <View
              style={
                [
                  {
                    "backgroundColor": "rgb(126, 72, 149)",
                    "borderRadius": 36,
                    "height": 72,
                    "width": 72,
                  },
                  {
                    "alignItems": "center",
                    "justifyContent": "center",
                  },
                  {},
                ]
              }
            >
              <Text />
            </View>
            <View
              style={
                [
                  {
                    "backgroundColor": "rgb(158, 55, 116)",
                    "borderRadius": 14,
                    "height": 28,
                    "width": 28,
                  },
                  {
                    "alignItems": "center",
                    "justifyContent": "center",
                  },
                  {
                    "alignItems": "center",
                    "borderRadius": 100,
                    "bottom": -10,
                    "height": 30,
                    "justifyContent": "center",
                    "position": "absolute",
                    "right": -4,
                    "width": 30,
                  },
                ]
              }
            >
              <Text />
            </View>
          </View>
        </View>
        <Text
          style={
            [
              {
                "textAlign": "left",
              },
              {
                "color": "rgba(28, 27, 31, 1)",
                "fontFamily": "System",
                "fontWeight": "400",
                "letterSpacing": 0,
              },
              {
                "writingDirection": "ltr",
              },
              {
                "color": "#414d63",
                "fontSize": 19,
                "fontWeight": "600",
                "marginTop": 20,
                "textAlign": "center",
              },
            ]
          }
        >
          Mi Perfil
        </Text>
        <Text
          style={
            [
              {
                "textAlign": "left",
              },
              {
                "color": "rgba(28, 27, 31, 1)",
                "fontFamily": "System",
                "fontWeight": "400",
                "letterSpacing": 0,
              },
              {
                "writingDirection": "ltr",
              },
              {
                "color": "#989898",
                "fontSize": 16,
                "marginTop": 5,
                "textAlign": "center",
              },
            ]
          }
        >
          Usuario MilaBot
        </Text>
      </View>
      <RCTScrollView>
        <View>
          <View
            style={
              {
                "paddingHorizontal": 24,
              }
            }
          >
            <Text
              style={
                [
                  {
                    "textAlign": "left",
                  },
                  {
                    "color": "rgba(28, 27, 31, 1)",
                    "fontFamily": "System",
                    "fontWeight": "400",
                    "letterSpacing": 0,
                  },
                  {
                    "writingDirection": "ltr",
                  },
                  {
                    "color": "#9e9e9e",
                    "fontSize": 12,
                    "fontWeight": "600",
                    "letterSpacing": 1.1,
                    "paddingVertical": 12,
                    "textTransform": "uppercase",
                  },
                ]
              }
            >
              Datos de Contacto
            </Text>
            <View
              accessibilityState={
                {
                  "busy": undefined,
                  "checked": undefined,
                  "disabled": true,
                  "expanded": undefined,
                  "selected": undefined,
                }
              }
              accessibilityValue={
                {
                  "max": undefined,
                  "min": undefined,
                  "now": undefined,
                  "text": undefined,
                }
              }
              accessible={true}
              collapsable={false}
              focusable={true}
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                [
                  false,
                  [
                    {
                      "paddingRight": 24,
                      "paddingVertical": 8,
                    },
                    [
                      {
                        "alignItems": "center",
                        "borderRadius": 8,
                        "flexDirection": "row",
                        "height": 50,
                        "justifyContent": "flex-start",
                        "marginBottom": 12,
                        "paddingLeft": 12,
                        "paddingRight": 12,
                      },
                      {
                        "backgroundColor": "#fff",
                      },
                    ],
                  ],
                ]
              }
            >
              <View
                style={
                  {
                    "flexDirection": "row",
                    "marginVertical": 6,
                    "width": "100%",
                  }
                }
              >
                <View
                  pointerEvents="box-none"
                  style={
                    [
                      {
                        "alignItems": "center",
                        "justifyContent": "center",
                      },
                      {
                        "alignSelf": "center",
                        "marginLeft": 16,
                        "marginRight": 0,
                      },
                    ]
                  }
                >
                  <Text />
                </View>
                <View
                  style={
                    [
                      {
                        "paddingLeft": 16,
                      },
                      {
                        "flexGrow": 1,
                        "flexShrink": 1,
                        "justifyContent": "center",
                      },
                      undefined,
                    ]
                  }
                  testID="undefined-content"
                >
                  <Text
                    numberOfLines={1}
                    selectable={false}
                    style={
                      [
                        {
                          "textAlign": "left",
                        },
                        {
                          "color": "rgba(28, 27, 31, 1)",
                          "fontFamily": "System",
                          "fontWeight": "400",
                          "letterSpacing": 0,
                        },
                        {
                          "writingDirection": "ltr",
                        },
                        [
                          {
                            "fontSize": 16,
                          },
                          {
                            "color": "rgba(28, 27, 31, 1)",
                          },
                          {
                            "color": "rgba(121, 116, 126, 1)",
                          },
                        ],
                      ]
                    }
                  >
                    Nombre
                  </Text>
                  <Text
                    numberOfLines={2}
                    onTextLayout={[Function]}
                    selectable={false}
                    style={
                      [
                        {
                          "textAlign": "left",
                        },
                        {
                          "color": "rgba(28, 27, 31, 1)",
                          "fontFamily": "System",
                          "fontWeight": "400",
                          "letterSpacing": 0,
                        },
                        {
                          "writingDirection": "ltr",
                        },
                        [
                          {
                            "fontSize": 14,
                          },
                          {
                            "color": "rgba(73, 69, 79, 1)",
                          },
                          {
                            "color": "rgba(121, 116, 126, 1)",
                          },
                        ],
                      ]
                    }
                  >
                    Mila
                  </Text>
                </View>
              </View>
            </View>
            <View
              accessibilityState={
                {
                  "busy": undefined,
                  "checked": undefined,
                  "disabled": true,
                  "expanded": undefined,
                  "selected": undefined,
                }
              }
              accessibilityValue={
                {
                  "max": undefined,
                  "min": undefined,
                  "now": undefined,
                  "text": undefined,
                }
              }
              accessible={true}
              collapsable={false}
              focusable={true}
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                [
                  false,
                  [
                    {
                      "paddingRight": 24,
                      "paddingVertical": 8,
                    },
                    [
                      {
                        "alignItems": "center",
                        "borderRadius": 8,
                        "flexDirection": "row",
                        "height": 50,
                        "justifyContent": "flex-start",
                        "marginBottom": 12,
                        "paddingLeft": 12,
                        "paddingRight": 12,
                      },
                      {
                        "backgroundColor": "#fff",
                      },
                    ],
                  ],
                ]
              }
            >
              <View
                style={
                  {
                    "flexDirection": "row",
                    "marginVertical": 6,
                    "width": "100%",
                  }
                }
              >
                <View
                  pointerEvents="box-none"
                  style={
                    [
                      {
                        "alignItems": "center",
                        "justifyContent": "center",
                      },
                      {
                        "alignSelf": "center",
                        "marginLeft": 16,
                        "marginRight": 0,
                      },
                    ]
                  }
                >
                  <Text />
                </View>
                <View
                  style={
                    [
                      {
                        "paddingLeft": 16,
                      },
                      {
                        "flexGrow": 1,
                        "flexShrink": 1,
                        "justifyContent": "center",
                      },
                      undefined,
                    ]
                  }
                  testID="undefined-content"
                >
                  <Text
                    numberOfLines={1}
                    selectable={false}
                    style={
                      [
                        {
                          "textAlign": "left",
                        },
                        {
                          "color": "rgba(28, 27, 31, 1)",
                          "fontFamily": "System",
                          "fontWeight": "400",
                          "letterSpacing": 0,
                        },
                        {
                          "writingDirection": "ltr",
                        },
                        [
                          {
                            "fontSize": 16,
                          },
                          {
                            "color": "rgba(28, 27, 31, 1)",
                          },
                          {
                            "color": "rgba(121, 116, 126, 1)",
                          },
                        ],
                      ]
                    }
                  >
                    Apellido
                  </Text>
                  <Text
                    numberOfLines={2}
                    onTextLayout={[Function]}
                    selectable={false}
                    style={
                      [
                        {
                          "textAlign": "left",
                        },
                        {
                          "color": "rgba(28, 27, 31, 1)",
                          "fontFamily": "System",
                          "fontWeight": "400",
                          "letterSpacing": 0,
                        },
                        {
                          "writingDirection": "ltr",
                        },
                        [
                          {
                            "fontSize": 14,
                          },
                          {
                            "color": "rgba(73, 69, 79, 1)",
                          },
                          {
                            "color": "rgba(121, 116, 126, 1)",
                          },
                        ],
                      ]
                    }
                  >
                    Bot
                  </Text>
                </View>
              </View>
            </View>
            <View
              accessibilityState={
                {
                  "busy": undefined,
                  "checked": undefined,
                  "disabled": true,
                  "expanded": undefined,
                  "selected": undefined,
                }
              }
              accessibilityValue={
                {
                  "max": undefined,
                  "min": undefined,
                  "now": undefined,
                  "text": undefined,
                }
              }
              accessible={true}
              collapsable={false}
              focusable={true}
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                [
                  false,
                  [
                    {
                      "paddingRight": 24,
                      "paddingVertical": 8,
                    },
                    [
                      {
                        "alignItems": "center",
                        "borderRadius": 8,
                        "flexDirection": "row",
                        "height": 50,
                        "justifyContent": "flex-start",
                        "marginBottom": 12,
                        "paddingLeft": 12,
                        "paddingRight": 12,
                      },
                      {
                        "backgroundColor": "#fff",
                      },
                    ],
                  ],
                ]
              }
            >
              <View
                style={
                  {
                    "flexDirection": "row",
                    "marginVertical": 6,
                    "width": "100%",
                  }
                }
              >
                <View
                  pointerEvents="box-none"
                  style={
                    [
                      {
                        "alignItems": "center",
                        "justifyContent": "center",
                      },
                      {
                        "alignSelf": "center",
                        "marginLeft": 16,
                        "marginRight": 0,
                      },
                    ]
                  }
                >
                  <Text />
                </View>
                <View
                  style={
                    [
                      {
                        "paddingLeft": 16,
                      },
                      {
                        "flexGrow": 1,
                        "flexShrink": 1,
                        "justifyContent": "center",
                      },
                      undefined,
                    ]
                  }
                  testID="undefined-content"
                >
                  <Text
                    numberOfLines={1}
                    selectable={false}
                    style={
                      [
                        {
                          "textAlign": "left",
                        },
                        {
                          "color": "rgba(28, 27, 31, 1)",
                          "fontFamily": "System",
                          "fontWeight": "400",
                          "letterSpacing": 0,
                        },
                        {
                          "writingDirection": "ltr",
                        },
                        [
                          {
                            "fontSize": 16,
                          },
                          {
                            "color": "rgba(28, 27, 31, 1)",
                          },
                          {
                            "color": "rgba(121, 116, 126, 1)",
                          },
                        ],
                      ]
                    }
                  >
                    Nro. de Identificación
                  </Text>
                  <Text
                    numberOfLines={2}
                    onTextLayout={[Function]}
                    selectable={false}
                    style={
                      [
                        {
                          "textAlign": "left",
                        },
                        {
                          "color": "rgba(28, 27, 31, 1)",
                          "fontFamily": "System",
                          "fontWeight": "400",
                          "letterSpacing": 0,
                        },
                        {
                          "writingDirection": "ltr",
                        },
                        [
                          {
                            "fontSize": 14,
                          },
                          {
                            "color": "rgba(73, 69, 79, 1)",
                          },
                          {
                            "color": "rgba(121, 116, 126, 1)",
                          },
                        ],
                      ]
                    }
                  >
                    12345678
                  </Text>
                </View>
              </View>
            </View>
            <View
              accessibilityState={
                {
                  "busy": undefined,
                  "checked": undefined,
                  "disabled": true,
                  "expanded": undefined,
                  "selected": undefined,
                }
              }
              accessibilityValue={
                {
                  "max": undefined,
                  "min": undefined,
                  "now": undefined,
                  "text": undefined,
                }
              }
              accessible={true}
              collapsable={false}
              focusable={true}
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                [
                  false,
                  [
                    {
                      "paddingRight": 24,
                      "paddingVertical": 8,
                    },
                    [
                      {
                        "alignItems": "center",
                        "borderRadius": 8,
                        "flexDirection": "row",
                        "height": 50,
                        "justifyContent": "flex-start",
                        "marginBottom": 12,
                        "paddingLeft": 12,
                        "paddingRight": 12,
                      },
                      {
                        "backgroundColor": "#fff",
                      },
                    ],
                  ],
                ]
              }
            >
              <View
                style={
                  {
                    "flexDirection": "row",
                    "marginVertical": 6,
                    "width": "100%",
                  }
                }
              >
                <View
                  pointerEvents="box-none"
                  style={
                    [
                      {
                        "alignItems": "center",
                        "justifyContent": "center",
                      },
                      {
                        "alignSelf": "center",
                        "marginLeft": 16,
                        "marginRight": 0,
                      },
                    ]
                  }
                >
                  <Text />
                </View>
                <View
                  style={
                    [
                      {
                        "paddingLeft": 16,
                      },
                      {
                        "flexGrow": 1,
                        "flexShrink": 1,
                        "justifyContent": "center",
                      },
                      undefined,
                    ]
                  }
                  testID="undefined-content"
                >
                  <Text
                    numberOfLines={1}
                    selectable={false}
                    style={
                      [
                        {
                          "textAlign": "left",
                        },
                        {
                          "color": "rgba(28, 27, 31, 1)",
                          "fontFamily": "System",
                          "fontWeight": "400",
                          "letterSpacing": 0,
                        },
                        {
                          "writingDirection": "ltr",
                        },
                        [
                          {
                            "fontSize": 16,
                          },
                          {
                            "color": "rgba(28, 27, 31, 1)",
                          },
                          {
                            "color": "rgba(121, 116, 126, 1)",
                          },
                        ],
                      ]
                    }
                  >
                    Email
                  </Text>
                  <Text
                    numberOfLines={2}
                    onTextLayout={[Function]}
                    selectable={false}
                    style={
                      [
                        {
                          "textAlign": "left",
                        },
                        {
                          "color": "rgba(28, 27, 31, 1)",
                          "fontFamily": "System",
                          "fontWeight": "400",
                          "letterSpacing": 0,
                        },
                        {
                          "writingDirection": "ltr",
                        },
                        [
                          {
                            "fontSize": 14,
                          },
                          {
                            "color": "rgba(73, 69, 79, 1)",
                          },
                          {
                            "color": "rgba(121, 116, 126, 1)",
                          },
                        ],
                      ]
                    }
                  >
                    mila.bot@me.com
                  </Text>
                </View>
              </View>
            </View>
            <View
              accessibilityState={
                {
                  "busy": undefined,
                  "checked": undefined,
                  "disabled": true,
                  "expanded": undefined,
                  "selected": undefined,
                }
              }
              accessibilityValue={
                {
                  "max": undefined,
                  "min": undefined,
                  "now": undefined,
                  "text": undefined,
                }
              }
              accessible={true}
              collapsable={false}
              focusable={true}
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                [
                  false,
                  [
                    {
                      "paddingRight": 24,
                      "paddingVertical": 8,
                    },
                    [
                      {
                        "alignItems": "center",
                        "borderRadius": 8,
                        "flexDirection": "row",
                        "height": 50,
                        "justifyContent": "flex-start",
                        "marginBottom": 12,
                        "paddingLeft": 12,
                        "paddingRight": 12,
                      },
                      {
                        "backgroundColor": "#fff",
                      },
                    ],
                  ],
                ]
              }
            >
              <View
                style={
                  {
                    "flexDirection": "row",
                    "marginVertical": 6,
                    "width": "100%",
                  }
                }
              >
                <View
                  pointerEvents="box-none"
                  style={
                    [
                      {
                        "alignItems": "center",
                        "justifyContent": "center",
                      },
                      {
                        "alignSelf": "center",
                        "marginLeft": 16,
                        "marginRight": 0,
                      },
                    ]
                  }
                >
                  <Text />
                </View>
                <View
                  style={
                    [
                      {
                        "paddingLeft": 16,
                      },
                      {
                        "flexGrow": 1,
                        "flexShrink": 1,
                        "justifyContent": "center",
                      },
                      undefined,
                    ]
                  }
                  testID="undefined-content"
                >
                  <Text
                    numberOfLines={1}
                    selectable={false}
                    style={
                      [
                        {
                          "textAlign": "left",
                        },
                        {
                          "color": "rgba(28, 27, 31, 1)",
                          "fontFamily": "System",
                          "fontWeight": "400",
                          "letterSpacing": 0,
                        },
                        {
                          "writingDirection": "ltr",
                        },
                        [
                          {
                            "fontSize": 16,
                          },
                          {
                            "color": "rgba(28, 27, 31, 1)",
                          },
                          {
                            "color": "rgba(121, 116, 126, 1)",
                          },
                        ],
                      ]
                    }
                  >
                    Celular
                  </Text>
                  <Text
                    numberOfLines={2}
                    onTextLayout={[Function]}
                    selectable={false}
                    style={
                      [
                        {
                          "textAlign": "left",
                        },
                        {
                          "color": "rgba(28, 27, 31, 1)",
                          "fontFamily": "System",
                          "fontWeight": "400",
                          "letterSpacing": 0,
                        },
                        {
                          "writingDirection": "ltr",
                        },
                        [
                          {
                            "fontSize": 14,
                          },
                          {
                            "color": "rgba(73, 69, 79, 1)",
                          },
                          {
                            "color": "rgba(121, 116, 126, 1)",
                          },
                        ],
                      ]
                    }
                  >
                    +569 12345678
                  </Text>
                </View>
              </View>
            </View>
          </View>
        </View>
      </RCTScrollView>
    </View>
  </View>
</RNCSafeAreaProvider>
`;
